src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+@for $i from 1 through 5 {
+  .stars.stars--#{$i} .stars__star:nth-child(-n + #{$i}) {
+    background-image: url(../images/star-active.svg);
```

Comment:
Similarly, use single quotes for URLs in CSS. Change the double quotes to single quotes in `background-image: url("../images/star-active.svg");`.